Nautilus/Patchers/KnownTechPatcher.cs

```diff
@@ -61,7 +61,7 @@ internal static void InitializePostfix()
             
         foreach (KnownTech.AnalysisTech tech in analysisTech)
         {
-            if (UnlockSound == null && tech.unlockSound != null && tech.techType == TechType.Lead)
+            if (UnlockSound == null && tech.unlockSound != null && tech.techType == TechType.CreepvinePiece)
```

Comment:
I'm surprised that went unnoticed, definitely a needed change. However I noticed that when I play the game, unlock sounds/notifications for KnownTech are hidden ENTIRELY. It occurs in vanilla, so might be a game bug or user error.

Reply:
True, but the patch below fixes the bug in subnautica:

```c#
// This patch fixes a bug in Subnautica that silences notifications like unlocked blueprints
[HarmonyPatch(typeof(Pickupable), nameof(Pickupable.Pickup))]
class Pickupable_Pickup_Patch
{
    static void Postfix(Pickupable __instance, bool events)
    {
        if (events)
            Analyze(__instance.GetTechType());
    }
}
```

I wasn't sure if I should suggest adding it to Nautilus. They may fix the bug in future.

Reply:
I'm sure it wasn't intentional. All the "creature discovered" & "new blueprint" sounds are still there, and calling `KnwonTech.Analyze` with `verbose` set to `true` makes it work, but only if you haven't already called it with `verbose` set to `false`. Unfortunately the code now _always_ calls it with `false` first, before calling it with `true`, which means it's _always_ silent.

Reply:
Exactly. Since we aren't sure about whether we should provide a fix via Nautilus or wait for UWE to fix it officially, I think this PR is good to go (as long as you are fine with it) and we can make a new issue report for that.

Reply:
Sounds Good.